docs/src/modules/components/AppSearch.js

```diff
@@ -67,10 +63,12 @@ function initDocsearch(userLanguage) {
       },
       // debug: true, // Set debug to true if you want to inspect the dropdown.
     });
+    // https://github.com/algolia/docsearch/issues/418
+    docsearchInput.setAttribute('aria-label', ariaLabel);
```

Comment:
This is the important change of the component, the other changes are about classes to hooks.